tests/Integration/Defer/DeferDBTest.php

```diff
@@ -252,7 +252,7 @@ public function itCanDeferNestedListFields(): void
 
         $deferredCompanies = $chunks[2];
         $this->assertCount(6, $deferredCompanies);
-        collect($deferredCompanies)->each(function (array $item) use ($companies): void {
+        Collection::make($deferredCompanies)->toBase()->each(function (array $item) use ($companies): void {
```

Comment:
It's faster to used `->toBase()` and we avoid confusion by importing `\Illuminate\Support\Collection` and `\Illuminate\Database\Eloquent\Collection`

Reply:
Or we just use `foreach`